hips/tiles/tests/test_tile.py

```diff
@@ -0,0 +1,34 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+from astropy.tests.helper import remote_data
+
+from ..tile import HipsTile
+from ..tile_meta import HipsTileMeta
+
+
+class TestHipsTile:
+    def setup(self):
+        self.hips_tile_meta_fits = HipsTileMeta(order=6, ipix=30889, format='fits')
+        self.hips_tile_meta_jpg = HipsTileMeta(order=6, ipix=30889, format='jpg')
+
+    @remote_data
+    def test_fetch_read_write(self):
```

Comment:
In tests that write files, you should always write to temp directories using the pytest `tmpdir` fixture:
https://docs.pytest.org/en/latest/tmpdir.html

You can find examples here: https://github.com/astropy/astropy/search?utf8=%E2%9C%93&q=tmpdir&type=
or here: https://github.com/gammapy/gammapy/search?utf8=%E2%9C%93&q=tmpdir&type=

I would suggest we always use this pattern to make filenames:
```
def test_read_write(self, tmpdir):
    filename = str(tmpdir / 'Npix30889.jpg')
    jpeg_tile.write(filename)
    jpeg_tile2 = Tile.read(filename)
```